requirements.txt

```diff
@@ -0,0 +1,22 @@
+# Please use pip to manage airflow dependencies.
+apache-airflow==1.10.13
+
+# Editable install with no version specified.
+google-cloud-bigquery
+pandas
+pyarrow
+pydantic<2
+pygsheets
+requests
+searchconsole
+StrEnum
```

Comment:
Is `StrEnum` required? Could we use the trick here? https://docs.python.org/3.8/library/enum.html#others

Reply:
I try to remove it.